package.nls.json

```diff
@@ -25,5 +25,6 @@
     "azFunc.pickProcess": "Pick Process",
     "azFunc.deploySubpathDescription": "The default subpath of a workspace folder to use when deploying.",
     "azFunc.installDotnetTemplates": "Install templates for the .NET CLI",
-    "azFunc.uninstallDotnetTemplates": "Uninstall templates for the .NET CLI"
+    "azFunc.uninstallDotnetTemplates": "Uninstall templates for the .NET CLI",
+    "azFunc.checkFunctionRuntimeDescription": "Specify whether the extension checks Azure Functions runtime."
```

Comment:
Suggestion: "Show a warning if your installed version of Azure Functions Core Tools is out-of-date."